### DISCUSSION TITLE
Import file formats

### DISCUSSION THREAD

**Discussion by janpernik** [mytourbook/mytourbook#1440] [Ideas]
Hi, I've started to use IpBike on mobile as sport tester (with USB ANT+ stick). It is able to export data in CSV fomat. 
Would be great to have option to import custom CSV file with some custom mapping. Example file attached
[IpBike_8_part.csv](https://github.com/user-attachments/files/18187579/IpBike_8_part.csv)

**Comment by wolfgang-ch** [comment]
There is already a CSV import https://mytourbook.sourceforge.io/mytourbook/index.php/documentation/get-tours/import-get-tours-supported-formats but not for this format.

Can you create a feature request (issue) for it
